NEXT_CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
  * Support for `uc_securable` resource in `databricks_app` ([#4767](https://github.com/databricks/terraform-provider-databricks/pull/4767))
  * Add `schema_id` to the output of `databricks_schema` resource ([#4773](https://github.com/databricks/terraform-provider-databricks/pull/4773)).
+ * Bump the Go SDK to v0.72.0 ([#4777](https://github.com/databricks/terraform-provider-databricks/pull/4777)).
```

Comment:
Changelog doesn't highlight the new resource - quality monitor v2

Reply:
+1, we need to implement changelog autogeneration for autogenerated resources in the provider still.